backend/clubs/views.py

```diff
@@ -5087,17 +5112,22 @@ def initiate_checkout(self, request, *args, **kwargs):
                 cart.save()
 
             # Place hold on tickets for 10 mins
-            holding_expiration = timezone.now() + datetime.timedelta(minutes=10)
-            tickets.update(
-                holder=self.request.user, holding_expiration=holding_expiration
+            self.place_hold_on_tickets(tickets)
```

Comment:
Do we place holds anywhere else in the viewset? Would prefer to keep this logic inside the view if not.